stories/demos/components/Formatters/ChildRowDeleteButton.tsx

```diff
@@ -1,17 +1,37 @@
-import React from 'react';
+import React, { useRef, useLayoutEffect } from 'react';
 
 export interface ChildRowDeleteButtonProps {
-  onDeleteSubRow: () => void;
+  isCellSelected: boolean;
   isDeleteSubRowEnabled: boolean;
+  onDeleteSubRow: () => void;
 }
 
-export function ChildRowDeleteButton({ onDeleteSubRow, isDeleteSubRowEnabled }: ChildRowDeleteButtonProps) {
+export function ChildRowDeleteButton({ isCellSelected, onDeleteSubRow, isDeleteSubRowEnabled }: ChildRowDeleteButtonProps) {
+  const iconRef = useRef<HTMLSpanElement>(null);
+  useLayoutEffect(() => {
+    if (!isCellSelected) return;
+    iconRef.current?.focus({ preventScroll: true });
+  }, [isCellSelected]);
+
+  function handleKeyDown(e: React.KeyboardEvent<HTMLSpanElement>) {
+    if (e.key === 'Enter') {
+      e.preventDefault();
+      onDeleteSubRow();
+    }
+  }
+
   return (
     <>
       <div className="rdg-child-row-action-cross" />
       {isDeleteSubRowEnabled && (
         <div className="rdg-child-row-btn" onClick={onDeleteSubRow}>
-          ❌
+          <span
```

Comment:
Why add another element?
Also I think if you just set `onClick={onDeleteSubRow}` instead of handling `onKeyDown` manually it might work just as well.

Reply:
Otherwise it looks like this

![image](https://user-images.githubusercontent.com/1452717/88819183-45cade00-d185-11ea-915f-805deb2ccfb2.png)